migrations/versions/b3ed927671bd_.py

```diff
@@ -0,0 +1,34 @@
+"""Add last_passed_on and description fields in regression_test
+
+Revision ID: b3ed927671bd
+Revises: a5183973c3e9
+Create Date: 2023-08-17 00:41:01.237549
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import mysql
```

Comment:
`mysql` is not used. Just a note, no change required as it is autogenerated file.